main_test.go

```diff
@@ -176,3 +176,28 @@ func TestDownloadWithContext_ErrorUserTimeout(t *testing.T) {
 		t.Error("expected channel closed, but did not get it")
 	}
 }
+
+func TestDownload_Chunks(t *testing.T) {
+	d := DefaultDownloader()
```

Comment:
How about explicitly setting the `ChunkSize`?

Reply:
Good. Then we can make less use of “magic numbers” and work with numbers our brain can parse easily (12 total bytes, chunk size of 5 etc).

Reply:
Bem melhor 87d92d9dca4fdfc41a68533f95bf0cb064358bf2 ✨

Reply:
> Good. Then we can make less use of “magic numbers” and work with numbers our brain can parse easily (12 total bytes, chunk size of 5 etc).

Exactly! 😄